### PR TITLE
Fix webhook validation

### PR DESCRIPTION
**Pull request by DimitarPetrov**

### CODE REVIEW

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Comment by DimitarPetrov** [comment]
reset-vm

**Comment by DimitarPetrov** [comment]
reset-vm

File: tests/director/tests/formation_api_test.go [hunk 1]
```diff
@@ -1727,11 +1727,11 @@ func TestFormationAssignments(stdT *testing.T) {
...
+		inputTemplateRuntime := "{\\\"ucl-formation-id\\\":\\\"{{.FormationID}}\\\",\\\"config\\\":\\\"{{ .ReverseAssignment.Value }}\\\",\\\"formation-assignment-id\\\":\\\"{{ .Assignment.ID }}\\\",\\\"items\\\":[{\\\"region\\\":\\\"{{ if .Application.Labels.region }}{{.Application.Labels.region}}{{ else }}{{.ApplicationTemplate.Labels.region}}{{ end }}\\\",\\\"application-namespace\\\":\\\"{{.ApplicationTemplate.ApplicationNamespace}}\\\",\\\"tenant-id\\\":\\\"{{.Application.LocalTenantID}}\\\",\\\"ucl-system-tenant-id\\\":\\\"{{.Application.ID}}\\\"}]}"
```
Comment (ivantenevvasilev) [inline tests/director/tests/formation_api_test.go:1730]: quotes on the config value are unnecessary

Comment (DimitarPetrov) [reply tests/director/tests/formation_api_test.go]: I just realized that and fixed it, however I don't understand why is that special processing needed. I was just going to ask you offline, so let's discuss.

**Comment by DimitarPetrov** [comment]
reset-vm

**Review by ivantenevvasilev — APPROVED** [review]

**Pull request merged**